Mindaugas Bankauskas CA about Docker (#158)
NoroffFEU/development-platforms: src/content/docs/containerzation/docker/docker-case-study.md

---
title: Docker
keywords: development platform, containerzation, package, docker

> tags: development platform, containerzation, package, docker

Reviewer: ```suggestion
tags: development platform, containerization, package, docker
```

---

> # Docker

Reviewer: You can remove this heading.

## Introduction

Docker is a tool that is used to automate the deployment of applications in lightweight containers so that applications can work efficiently in different environments in isolation.\

> Docker is a way to package software so it can run on any hardware.

Reviewer: ```suggestion

Docker is a way to package software so it can run on any hardware.
```

<img src="https://pbs.twimg.com/media/FPKqqiFX0AMRBu4?format=png&name=small" alt="It works on my machine, then we will ship your machine. That is how Docker was born." width="50%" height="auto">

> Docker fixes the problem, where the developer code works on his hardware, but does not work on other machines.\

Reviewer: ```suggestion
Docker fixes the problem where the developer's code works on their hardware, but does not work on other machines.
```

> So you will never run into this situation with your software testers:

Reviewer: ```suggestion

So you will never run into this situation with your software testers:
```
